core/src/main/scala/org/apache/spark/storage/BlockObjectWriter.scala

```diff
@@ -20,10 +20,17 @@ package org.apache.spark.storage
 import java.io.{BufferedOutputStream, FileOutputStream, File, OutputStream}
 import java.nio.channels.FileChannel
 
-import org.apache.spark.Logging
-import org.apache.spark.serializer.{SerializationStream, Serializer}
-import org.apache.spark.executor.ShuffleWriteMetrics
+import org.apache.hadoop.mapreduce.security.TokenCache
 
+import com.intel.chimera.{CipherSuite, CryptoOutputStream, CryptoCodec}
+
+import org.apache.spark.crypto.CommonConfigurationKeys.SPARK_ENCRYPTED_INTERMEDIATE_DATA_BUFFER_KB
+import org.apache.spark.crypto.CommonConfigurationKeys.SPARK_SHUFFLE_TOKEN
+import org.apache.spark.crypto.CommonConfigurationKeys.DEFAULT_SPARK_ENCRYPTED_INTERMEDIATE_DATA_BUFFER_KB
+import org.apache.spark.deploy.SparkHadoopUtil
+import org.apache.spark.executor.ShuffleWriteMetrics
+import org.apache.spark.serializer.{SerializationStream, Serializer}
+import org.apache.spark.{Logging,SparkConf}
```

Comment:
nit: this is out of order (should be at the top of this group), also needs an empty line afterwards.